msgcntr/messageforums-app/src/webapp/css/msgcntr.css

```diff
@@ -681,3 +681,12 @@ div.msgcntr-profile-qtip {
 	cursor: pointer;
 	font-size: 20px !important;
 }
+
+.closeDate, .openDate {
+	width: auto !important;
+	margin-left: 10px !important;
```

Comment:
can you do this without using important? For example, can your CSS rule be more specific?